meson.build

```diff
@@ -44,6 +44,37 @@ if get_option('enable-tflite')
   cdata.set('HAVE_TFLITE', 1)
 endif
 
+# Define library dependencies for TensorRT support
+if get_option('enable-tensorrt')
+
+  # CUDA versions supported
+  cuda_versions = [
+    '10.0',
+  ]
+
+  # Find CUDA version
+  cuda_ver = ''
```

Comment:
Why we are not looking for a specific cuda version?

Reply:
I implemented a mechanism to add supported CUDA versions, and Meson will look for any of them installed in the system